packages/web/src/sideEffects/localFs/index.js

```diff
@@ -3,9 +3,41 @@ const { callbackToObservable } = require('@jscad/core').observableUtils
 
 const { changedFiles, flattenFiles } = require('./utils')
 
+const localFsOptions = require('../../../data/localFsOptions.json')
+
 const makeLocalFsSideEffect = async (params) => {
+  console.log('params', JSON.parse(JSON.stringify(params)))
```

Comment:
GAG. I didn't see this last review. Can you comment out this debug line? thanks

Reply:
ok